fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionManager.java

```diff
@@ -0,0 +1,103 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.lake.compaction;
+
+import com.clearspring.analytics.util.DoublyLinkedList;
+import com.clearspring.analytics.util.ListNode2;
+import com.clearspring.analytics.util.Lists;
+import com.google.common.base.Preconditions;
+import com.starrocks.common.Config;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CompactionManager {
+    private static final Logger LOG = LogManager.getLogger(CompactionManager.class);
+
+    private final Map<PartitionIdentifier, ListNode2<PartitionStatistics>> statisticsMap = new HashMap<>();
```

Comment:
Why use `ListNode2<T>`, not directly use `PartitionStatistics`?

Reply:
updated